integration_tests/src/main/python/json_test.py

```diff
@@ -569,26 +654,18 @@ def test_from_json_struct_decimal():
     # boolean
     "(true|false)"
 ])
-@pytest.mark.parametrize('date_format', [
-    pytest.param("", marks=pytest.mark.allow_non_gpu(*non_utc_project_allow)),
-    pytest.param("yyyy-MM-dd", marks=pytest.mark.allow_non_gpu(*non_utc_project_allow)),
-    # https://github.com/NVIDIA/spark-rapids/issues/9667
-    pytest.param("dd/MM/yyyy", marks=pytest.mark.allow_non_gpu('ProjectExec')),
-])
-@pytest.mark.parametrize('time_parser_policy', [
-    pytest.param("LEGACY", marks=pytest.mark.allow_non_gpu('ProjectExec')),
```

Comment:
I lost the `marks=pytest.mark.allow_non_gpu(*non_utc_project_allow))` from the date formats when I improved the test to test with all supported date formats, so I added `@allow_non_gpu(*non_utc_project_allow)` instead, but this caused the marks on `LEGACY` to be ignored and caused the test to fail.

The following test specifically tests that `LEGACY` falls back to the CPU, so removed `LEGACY` from this test.